### DISCUSSION TITLE
Is there a way to space nodes between each other ?

### DISCUSSION THREAD

**Discussion by hugo73cr** [sgratzl/chartjs-chart-graph#108] [Q&A]
Hello, is there a way to add a sort of padding to the nodes to avoid the overlaping ?
![image](https://github.com/sgratzl/chartjs-chart-graph/assets/167103815/f9ca5d8c-9f3c-4027-8705-75fe0d5239d1)
Thanks !

**Comment by sgratzl** [comment]
looks like your node size is too big

**Reply by hugo73cr** [reply]
I wanted to keep this node size, otherwise is there a way to increment the Y property of a node every two nodes ? I draw my graph without specify X and Y, with the force directed

**Reply by sgratzl** [reply]
see also 
https://github.com/sgratzl/chartjs-chart-graph/blob/5ed657f434e0d6259d4b5339041e5b6c9f90ab99/src/controllers/ForceDirectedGraphController.ts#L60-L66
in the controller options, which will allow to configure the collide force: https://d3js.org/d3-force/collide
